include/pybind11/attr.h

```diff
@@ -12,7 +12,7 @@
 
 #include "cast.h"
 
-NAMESPACE_BEGIN(pybind11)
+NAMESPACE_BEGIN(PYBIND11_NAMESPACE)
```

Comment:
I'm against that, for selfish reasons :). I have several other projects that use the ``NAMESPACE_BEGIN`` macro, where that behavior may not be intended.